.cargo/config.toml

```diff
@@ -1,19 +1,45 @@
-#[target.xtensa-esp32s3-none-elf]
+[alias]
+build-esp32   = "build --release --target xtensa-esp32-none-elf --features esp32"
+build-esp32c2 = "build --release --target riscv32imc-unknown-none-elf --features esp32c2"
+build-esp32c3 = "build --release --target riscv32imc-unknown-none-elf --features esp32c3"
+#build-esp32c5 = "build --release --target riscv32imac-unknown-none-elf --features esp32c5"
+build-esp32c6 = "build --release --target riscv32imac-unknown-none-elf --features esp32c6"
+build-esp32s2 = "build --profile esp32s2 --target xtensa-esp32s2-none-elf --features esp32s2"
+build-esp32s3 = "build --release --target xtensa-esp32s3-none-elf --features esp32s3"
+#build-esp32p4 = "build --release --target riscv32imac-unknown-none-elf --features esp32p4"
```

Comment:
Does this project makes sense for P4? It doesn't have any connectivity (wifi/ble..)